jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/ServletPathSpec.java

```diff
@@ -291,4 +343,28 @@ public boolean matches(String path)
                 return false;
         }
     }
+
+    public static class ServletMatchedPath implements MatchedPath
```

Comment:
There is nothing "Servlet" about this.   Just add a static method on `MatchedPath` so you can say `MatchedPath.from(pathMatch, pathInfo)`

Reply:
Done.